jobs/config.json

```diff
@@ -10114,6 +10172,28 @@
       "sig-gcp"
     ]
   },
+  "ci-kubernetes-soak-gci-gce-beta": {
+    "args": [
+      "--down=false",
+      "--env=DOCKER_TEST_LOG_LEVEL=--log-level=warn",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=ci/k8s-beta",
+      "--gcp-master-image=gci",
+      "--gcp-node-image=gci",
+      "--gcp-project=k8s-jkns-gce-gci-soak-1-5", // TODO(@cjwagner) What project can I use here?
```

Comment:
you can use `k8s-jkns-gce-soak-1-6` which I deleted the job from https://github.com/kubernetes/test-infra/pull/6538

Reply:
you can't comment like this in JSON, this is why the tests are failing.
in fact, you can't comment in JSON at all technically, but you can put a bogus "_comment": "comment text" field before the args.

xref: #4794